lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -270,6 +270,7 @@ sub archive ($self) {
         die "Unable to copy '$normal_result_dir' to '$archived_result_dir': $error";
     }
 
+    $signal_guard->retry(0) if $signal_guard;
```

Comment:
What's the purpose of this method call? I only see it used in a test, but `OpenQA::Task::Job::ArchiveResults` doesn't pass along a guard. 🤔

Reply:
Oh, it should pass it. I'll fix that, good catch.